docs/Number theory/Egyptian Fraction.md

```diff
@@ -0,0 +1,144 @@
+---
+id: egyptian-fraction
+title: Egyptian Fraction
+sidebar_label: Introduction to Egyptian Fractions
+description: 'Egyptian Fractions represent a fraction as a sum of distinct unit fractions.'
+tags: [dsa, number-theory, fractions]
+---
+
+### Overview:
+
+An Egyptian Fraction is a representation of a fraction as a sum of distinct unit fractions. For example, the fraction \( \frac{2}{3} \) can be expressed as \( \frac{1}{2} + \frac{1}{6} \). The challenge is to find such representations efficiently.
+

```

Comment:
replace `\( \frac{2}{3} \)` with  `$\frac{2}{3}$` and `\( \frac{1}{2} + \frac{1}{6} \)` with `$\frac{1}{2} + \frac{1}{6}$`